ROADMAP.md

```diff
@@ -5,18 +5,19 @@ The features and themes included in each milestone are optimistic in the sense t
 Community and contributor involvement is vital for successfully implementing all desired items for each release.
 We hope that the items listed below will inspire further engagement from the community to keep Rook progressing and shipping exciting and valuable features.
 
-Any dates listed below and the specific issues that will ship in a given milestone are subject to change but should give a general idea of what we are planning.
+Any dates listed below and the specific issues that will ship in a given milestone are subject to change but give a general idea of what we are planning.
 See the [GitHub project boards](https://github.com/rook/rook/projects) for the most up-to-date issues and their status.
 
-## Rook Ceph 1.17
+## Rook Ceph 1.18
 
-The following high level features are targeted for Rook v1.17 (April 2025). For more detailed project tracking see the [v1.17 board](https://github.com/orgs/rook/projects/7).
+The following high level features are targeted for Rook v1.18 (August 2025). For more detailed project tracking see the [v1.18 board](https://github.com/orgs/rook/projects/8).
 
+* Rotate cephx keyrings automatically [#15904](https://github.com/rook/rook/issues/15904)
 * Allow Rook operator to run in multiple namespaces for improved multi-cluster reconcile [#15014](https://github.com/rook/rook/issues/15014)
 * Replace a single OSD when a metadataDevice is configured with multiple OSDs [#13240](https://github.com/rook/rook/issues/13240)
 * CSI Driver
   * Enable the stable Ceph-CSI operator by default, after it is declared stable [#14766](https://github.com/rook/rook/issues/15271)
-  * Integrate Ceph-CSI [v3.14](https://github.com/ceph/ceph-csi/milestone/23)
+  * Integrate Ceph-CSI [v3.15](https://github.com/ceph/ceph-csi/milestones)
```

Comment:
Is this detail/aspect still relevant after switching over to ceph-csi-operator?

Reply:
It's true that we will pick up the CSI driver 3.15 by default. But that's a good question how exactly that will work with csi operator, not sure that detail.